packages/asset-swapper/src/utils/market_operation_utils/path_optimizer.ts

```diff
@@ -17,7 +17,7 @@ import { DexSample, ERC20BridgeSource, FeeSchedule, Fill, FillData } from './typ
 // tslint:disable: prefer-for-of custom-no-magic-numbers completed-docs no-bitwise
 
 const RUN_LIMIT_DECAY_FACTOR = 0.5;
-const RUST_ROUTER_NUM_SAMPLES = 200;
+const RUST_ROUTER_NUM_SAMPLES = 14;
```

Comment:
So what do the curves look like with this workaround?

Reply:
We match the output splits with the sampling spilts such that there is no need to do interpolation. Is my understanding correct?

Reply:
The curve will basically be lines from 0 through our sampled points.
![image](https://user-images.githubusercontent.com/5450545/146962119-67ddc4b8-e9c4-4f51-a16d-1eecfd46c462.png)

With this configuration, the delta between interpolated points when creating the curve will be exactly the delta between our sample inputs. So we will basically just reuse the input/output of the samples instead of trying to create a smoother curve.